app/routes/events/view/index.js

```diff
@@ -9,7 +9,9 @@ export default Route.extend({
       event    : eventDetails,
       sponsors : eventDetails.query('sponsors', {
         'page[size]': 10
-      })
+      }),
+      orderStat  : eventDetails.query('orderStatistics', {}),
+      ticketStat : eventDetails.query('tickets', {})
```

Comment:
@niranjan94 The order stats API for ticket has the endpoint as `/v1/tickets/{ticket_id}/order-statistics`, now if we try to include in query as using `include` it throws an error because there is no such model on server.